doc/manipulating_visualizing/manipulating_images.rst

```diff
@@ -149,22 +161,22 @@ downloaded, a single line is needed to load it.
 Niimg-like objects
 -------------------
 
-Often, nilearn functions take as input parameters what we call
-"Niimg-like objects:
+As a baseline, nilearn functions take as input argument what we call
+"Niimg-like objects":
 
-**Niimg:** A Niimg-like object can either be:
+**Niimg:** A Niimg-like object can be one of the following:
 
-  * a file path to a Nifti or Analyse image
-  * any object exposing ``get_data()`` and ``get_affine()`` methods, for
-    instance a ``Nifti1Image`` from nibabel_.
+  * A string variable with a file path to a Nifti or Analyse image
+  * Any object exposing ``get_data()`` and ``get_affine()`` methods, typically
+    a ``Nifti1Image`` from nibabel_.
 
-**Niimg-4D:** Similarly, some functions require 4-dimensional Nifti-like
-data, which we call Niimgs, or Niimg-4D. Accepted inputs are then:
+**Niimg-4D:** Similarly, some functions require sets of Nifti-like
```

Comment:
For me, **sets of Nifti-like** means that we have several which has no sense since we don't know what is the dimension of the nifti-like.